accepted/PSR-12-extended-coding-style-guide.md

```diff
@@ -882,8 +888,15 @@ All operators not described here are left undefined.
 
 ### 6.1. Unary operators
 
-The increment/decrement operators MUST NOT have any space between
-the operator and operand.
+The increment/decrement operators, the logical not operator, the conversion 
+and negation operators MUST NOT have any space between the operator and operand.
+~~~php
+$value++;
+--$value;
+$a = !$b;
+$c = +$d;
+$e = -$f;
```

Comment:
Does it make sense the syntax
```php
$c = +$d;
$e = -$f;
```

?

Reply:
Those are unary PHP operators.
They can actually be confused with arithmetic operations.

`
$a = -$b; // opposite value in |N or |R sets in maths
$a = +$b; // identity convert the variable
$a = --$b; // convert and substract before using the value
$a = ++$b; // convert and sum before using the value
$a = 1 + $b; // convert and add 1
$a = 1 +$b; // still adds :)

$c = 1.0;
var_dump(+$c); // float(1)
var_dump(-$c); // float(-1)
var_dump(+-$c); // float(-1)
var_dump(-+$c); // float(-1)
`